.github/workflows/deploy-master.yaml

```diff
@@ -0,0 +1,19 @@
+name: Deploy Master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.12.12
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
```

Comment:
Since I don't have admin access on `dvc.org`, I wasn't able to add my account API key to the secrets. @iterative/websites, anyone with admin access that can help me add this?